lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -185,11 +186,18 @@ def add_tunnel_flows(self, precedence:int, i_teid:int,
         # Install Downlink Tunnel
         actions = []
         if ue_ip_adr:
+            # Add flow for LOCAL port
+            if ue_ip_adr.version == IPAddress.IPV6:
+                match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IPV6,
+                                   in_port=self._uplink_port,
```

Comment:
Can we do something like this? Put the get_eth_ip_match function into policy converters.py (see the get_ue_ip_match_args function f.e)
```
   eth_ip_match = get_eth_ip_match(ue_ip_addr)

    return MagmaMatch(in_port...... , **eth_ip_match)

```

Reply:
Yes.. changes done.